src/utils.js

```diff
@@ -65,3 +66,34 @@ export const npm = {
     }[npm.manager]);
   }
 };
+
+export const codeshift = {
+  get path() {
+    let value = resolve(__dirname, '../.codeshift');
+    Object.defineProperty(codeshift, 'path', { value });
+    return value;
+  },
+
+  async install(lang, bin, install) {
+    bin = resolve(__dirname, '../.codeshift', lang, bin);
+
+    if (!existsSync(bin)) {
+      await install();
+    }
+
+    codeshift[lang].bin = bin;
+    return bin;
+  },
+  run: async (lang, args) => run(await codeshift[lang].install(), args),
```

Comment:
Small style thing — newline after the `},`

Reply:
I also wouldn't be opposed to being consistent with the other async signature for install (async before the property name).

```js
async run(lang, args) {
  return run(await codeshift[lang].install(), args);
},
```